node/lndhub.py

```diff
@@ -0,0 +1,81 @@
+import json
+import logging
+import qrcode
+import time
+from typing import Tuple
+
+import config
+
+
+class lndhub:
+    def __init__(self, node_config: dict) -> None:
+        from blue_wallet_client import BlueWalletClient
+
+        self.config = node_config
+        self.is_onchain = False
+
+        for i in range(config.connection_attempts):
+            try:
+                logging.info("Attempting to initialize LNDHub client...")
+                self.lndhub = BlueWalletClient(
+                    bluewallet_login=self.config["bw_login"],
```

Comment:
This should be `"bw_username"` to match config.toml, then it works great. Awesome job!

Reply:
oops

Reply:
Code used `bw_login` everywhere, changed `config.toml` to match.